tyk-pro/configs/tyk_sink.conf

```diff
@@ -0,0 +1,36 @@
+{
+    "listen_port": 9090,
+    "healthcheck_port": 8181,
+    "server_options": {
+        "use_ssl": false,
+        "certificate": {
+            "cert_file": "certificate.pem",
+            "key_file": "key.pem"
+        },
+        "min_version": 771
+    },
+    "storage": {
+        "type": "redis",
+        "host": "",
+        "port": 6379,
+        "username": "",
+        "password": "",
+        "database": 0,
+        "optimisation_max_idle": 0,
+        "optimisation_max_active": 0,
+        "enable_cluster": false,
+        "hosts": null,
+        "redis_use_ssl": false,
+        "redis_ssl_insecure_skip_verify": false
+    },
+    "analytics": {
+        "mongo_url": ""
+    },
+    "hash_keys": true,
+    "session_timeout": 0,
+    "forward_analytics_to_pump": false,
```

Comment:
This should be configurable. The chart includes a pump deployment, I think it makes sense to forward in most cases.